### DISCUSSION TITLE
[Feature] can i send feishu alarm to different user group according to servicename

### DISCUSSION THREAD

**Discussion by peterpanzsy** [apache/skywalking#9696] [Ideas]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

i want to send alarm to different users according to service，or send notification to service owners.

### Use case

can I config multi feishu webhooks, matching different users，and different service chooses different feishu webhook to send message

### Related issues

alarm

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
No, you can't. This should be a discussion, not a feature. 
In SkyWalking, feature is only tracking issues having ppl working on.

**Reply by peterpanzsy** [reply]
what does ppl mean?

**Reply by wu-sheng** [reply]
`ppl` == `people` Sorry.

**Comment by wu-sheng** [comment]
Management is your own responsibility, if you want to a complex ownership module, the gRPC/HTTP hook should be a better option.

**Reply by peterpanzsy** [reply]
OK, I see. So I'd better implement a service to manage service-owners and call feishu
